src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -769,6 +780,18 @@ class ScallopGobraConfig(arguments: Seq[String], isInputOptional: Boolean = fals
       Right(())
     }
   }
+  
+  // `disableSetAxiomatization` can only be provided when using a silicon-based backend
+  // since, at the time of writing, we rely on Silicon's setAxiomatizationFile for the
+  // implementation
```

Comment:
assuming that the axiomatisations in carbon define the same symbols, why can't we use this approach with carbon?

Reply:
I don't think Carbon has an equivalent to this flag. Instead, the set axiomatization seems to be [hardcoded](https://github.com/viperproject/carbon/blob/2e12ccdd636f8e51de2d1719338613dd98bc6dd5/src/main/scala/viper/carbon/modules/impls/sequence_axioms/SetAxiomatization.scala). Additionally, it appears to be written in Boogie (instead of Viper), and also contains axiomatizations beyond sets - namely multisets.